lib/qbo_api/raise_http_exception.rb

```diff
@@ -48,6 +51,8 @@ def error_body(body)
       else
         nil
       end
+    rescue => _error
+      nil
```

Comment:
👎 on silently swallowing exceptions

Reply:
I had the specific error in there previously, but I thought that there might be other error types raised since there are many gems that patch the JSON class. Do you still think the errors should be specified? I'd be happy to revert it to that.

Reply:
If we *have* to swallow the error here, which I'm not confident we should, then we should at least add an method like

```ruby
rescue => error
  handle_error_body_parsing_exception(body, error)
end

# override this method in FaradayMiddleware:: RaiseHttpException to do something with the error
def handle_error_body_parsing_exception(body, error)
  QboApi.logger.error "Failed to parse body=#{body.inspect} error=#{error.inspect}"
  nil
end
```

Reply:
@bf4 Probably should swallow and ur approach looks good.